domains/chase.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "MrStuffmaker",
+           "email": "96948850+MrStuffmaker@users.noreply.github.com",
+           "discord": "213586333677912064"
+        },
+    
+        "record": {
+            "A": ["109.71.253.24"]
```

Comment:
IP seems to redirect to web server;s default page, please give a preview of your website